# Average Order Value (AOV)


## An e-commerce platform tests two pricing strategies on different user groups to check if the average order value changes.

In [21]:
from scipy.stats import ttest_ind

In [15]:
# Group A (Old Pricing): 20 users, orders
group_a = [50, 60, 45, 70, 80, 90, 100, 75, 85, 95, 60, 55, 67, 77, 88, 92, 110, 120, 85, 95]

# Group B (New Pricing): 20 users, orders
group_b = [55, 65, 50, 75, 85, 95, 105, 80, 90, 100, 65, 60, 70, 80, 90, 95, 115, 125, 90, 100]

HO: No significant difference in AOV between groups

H1: AOV is different

In [23]:
t_stat, p_value = ttest_ind(a=group_a, b=group_b, equal_var=False)

In [25]:
print(f'T-statistic: {t_stat:.4f}')
print(f'P-value: {p_value:.4f}')

T-statistic: -0.7217
P-value: 0.4749


T-statistic = -0.7217: Indicates the direction and magnitude of the difference (negative means Group A has a lower mean than Group B, but the difference is small).

p-value = 0.4749: This is way above 0.05, meaning we fail to reject the null hypothesis (H₀). There is no strong evidence that the new pricing strategy affects AOV.